velox/expression/fuzzer/ExpressionFuzzerVerifier.cpp

```diff
@@ -109,7 +110,10 @@ ExpressionFuzzerVerifier::ExpressionFuzzerVerifier(
       referenceQueryRunner_{
           options_.expressionFuzzerOptions.referenceQueryRunner} {
   filesystems::registerLocalFileSystem();
+  connector::registerConnectorFactory(
+      std::make_shared<connector::hive::HiveConnectorFactory>());
   exec::test::registerHiveConnector({});
+  dwrf::registerDwrfWriterFactory();
```

Comment:
is this related to the feature we are adding?

Reply:
Ditto.